WordPress/Classes/ViewRelated/Basic Profile/BasicUserProfileViewController.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+
+final class BasicUserProfileViewController: UIViewController, DrawerPresentable, StoryboardLoadable {
+    static var defaultStoryboardName: String = "BasicUserProfile"
+
+    var expandedHeight: DrawerHeight = DrawerHeight.intrinsicHeight
+
+    let allowsUserTransition = false
+
+    @IBOutlet private weak var gravatarImageView: CircularImageView!
+    @IBOutlet private weak var nameLabel: UILabel!
+    @IBOutlet private weak var usernameLabel: UILabel!
+    @IBOutlet private weak var locationLabel: UILabel!
+    @IBOutlet private weak var aboutMeLabel: UILabel!
+    @IBOutlet private weak var contentView: UIView!
+    @IBOutlet private weak var sitesTableView: UITableView!
+
+    var viewModel: BasicUserProfileViewModel!
```

Comment:
For most of the time I strongly agree with you, but with this case the controller is meaningless without a view model and I wanted to emphasize that. Also we can clear lots of unwrapping from the code. But still if this is against the convention, I can update?

Reply:
I think the most important thing is to avoid crashes, and I totally agree that the `viewModel` is necessary. The ideal scenario would be to have it injected at initialization time, but I realize that this might require a bit of changes because we are using storyboards, and I don't want to over complicate your trial project. If you'd agree, we could proceed as follows:

- For now, let's just do the explicit unwrapping instead of the implicit: thinking about the impacts on the app, is certainly better if a profile page is shown blank or with missing parts than the app crashing after trying to view a profile

- In the future, we could refactor this `viewController` in order to have the `viewModel` injected at initialization time

What do you think?

Reply:
Sure, sounds great. Will update that with next PR. 👍